### PR TITLE
Feat bundle files

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

**Comment by doktordirk** [comment]
@RWOverdijk no lables. can review

File: README.md [hunk 1]
```diff
@@ -17,13 +17,17 @@ This library is used directly by applications only.
 
 This library can be used in the **browser** only.
 
+## Important note
+
+This plugin is now registered with npm and jspm as `aurelia-notification`. This simplifies the installation. With this change `aurelia-notification` will reference to [humane-js](http://wavded.github.io/humane-js/) as `humane-js` only (loosing the `wavded/..` prefix). You will need to update your project's package.json accordingly to prevent forks.
```
Comment (RWOverdijk) [inline README.md]: Simplify like other PRs

File: README.md [hunk 2]
```diff
@@ -17,13 +17,17 @@ This library is used directly by applications only.
...
+
 ## Installation
 
 Installing this module is fairly simple.
 
-Run `jspm install github:spoonx/aurelia-notification` from your project root.
+Run `jspm install aurelia-notification` from your project root.
 
-Either set your own styles or install humane.js with `jspm install github:wavded/humane-js` to have some default styles to choose from.
+Either set your own styles or install humane-js with `jspm install humane-js` to have some default styles to choose from.
```
Comment (RWOverdijk) [inline README.md]: I'm not sure if this works.

**Comment by RWOverdijk** [comment]
:+1:

**Comment by doktordirk** [comment]
ups. had included the name change. not much harm though hopefully

**Pull request merged**